### PR TITLE
Okapi 796 reduce decoding module descriptors

### PR DESCRIPTION
**Pull request by adamdickmeiss**

### CODE REVIEW

File: okapi-core/src/main/java/org/folio/okapi/managers/ModuleManager.java [hunk 1]
```diff
@@ -38,6 +38,7 @@
   private String mapName = "modules";
   private LockedTypedMap1<ModuleDescriptor> modules
     = new LockedTypedMap1<>(ModuleDescriptor.class);
+  private Map<String,ModuleDescriptor> enabledModules = new HashMap<>();
```
Comment (julianladisch) [inline okapi-core/src/main/java/org/folio/okapi/managers/ModuleManager.java]: Is this a cache? If yes, can you rename it to `enabledModulesCache`?

Comment (julianladisch) [inline okapi-core/src/main/java/org/folio/okapi/managers/ModuleManager.java]: Is the cache pruned when ModuleManager.update or ModuleManager.delete is called, and are there unit tests for update and delete?

File: okapi-core/src/main/java/org/folio/okapi/managers/ProxyService.java [hunk 2]
```diff
@@ -685,7 +685,7 @@ private void proxyRequestResponse10(Iterator<ModuleInstance> it,
 
   private void copyHeaders(HttpClientRequest cReq, RoutingContext ctx, ModuleInstance mi) {
     int sz = 0;
-    int limit = 5; // all headers dumped
+    int limit = 2000; // all headers dumped
```
Comment (julianladisch) [inline okapi-core/src/main/java/org/folio/okapi/managers/ProxyService.java]: Can you also change

    if (sz > limit) {

to

    if (sz > limit && logger.isInfoEnabled()) {

and remove the unused StringBuilder from dumpHeaders?

File: okapi-core/src/main/java/org/folio/okapi/managers/ModuleManager.java (hunk 1 above)
Comment (adamdickmeiss) [reply okapi-core/src/main/java/org/folio/okapi/managers/ModuleManager.java]: yep

Comment (adamdickmeiss) [reply okapi-core/src/main/java/org/folio/okapi/managers/ModuleManager.java]: In reality not needed, but I'll just do it. MDs are immutable and probably MD PUT will be removed soon.

File: okapi-core/src/main/java/org/folio/okapi/managers/ProxyService.java (hunk 2 above)
Comment (adamdickmeiss) [reply okapi-core/src/main/java/org/folio/okapi/managers/ProxyService.java]: Yep. Will do

**Pull request merged**